src/routes.php

```diff
@@ -5,10 +5,19 @@
 
 // Routes
 
-$app->get('/[{name}]', function (Request $request, Response $response, array $args) {
-    // Sample log message
-    $this->logger->info("Slim-Skeleton '/' route");
+$app->get('/', function (Request $request, Response $response, array $args) {
 
+    $args['farm'] = 'hello';
     // Render index view
     return $this->renderer->render($response, 'index.phtml', $args);
 });
+
+
+$app->post('/api/login', function (Request $request, Response $response, array $args) {
```

Comment:
Can you remove this for merge with story?

Reply:
it was used for testing and we have now removed it.